controllers/drcluster_controller.go

```diff
@@ -277,6 +272,44 @@ func filterDRClusterMCV(mcv *viewv1beta1.ManagedClusterView) []ctrl.Request {
 	}
 }
 
+func filterDRClusterSecret(reader client.Reader, secret *corev1.Secret) []ctrl.Request {
+	log := ctrl.Log.WithName("filterDRClusterSecret").WithName("Secret")
+
+	drcusters := &ramen.DRClusterList{}
+	if err := reader.List(context.TODO(), drcusters); err != nil {
+		log.Error(err, "Failed to get list of DRCluster resources")
```

Comment:
This is shorter and more correct (since the actual api is List)

    Failed to list DRClusters

Do we want to include the secret being filtered in the error?

Reply:
Message changed to shorter one. Secret name is not related to the error, so...